packages/thirdweb/src/chainsaw/endpoints/getLatestBlockNumber.ts

```diff
@@ -0,0 +1,67 @@
+import type { ThirdwebClient } from "../../client/client.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import type { ChainsawResponse } from "../types.ts";
+import { getLatestBlockNumberEndpoint } from "../urls.js";
+
+export type GetLatestBlockNumberParams = {
+  /**
+   * A client is the entry point to the thirdweb SDK. It is required for all other actions.
+   *
+   * You can create a client using the `createThirdwebClient` function.
+   * Refer to the [Creating a Client](https://portal.thirdweb.com/typescript/v5/client) documentation for more information.
+   *
+   */
+  client: ThirdwebClient;
+  /**
+   * Chain ID
+   */
+  chainId: number;
+};
+
+export type GetLatestBlockNumberResult = {
+  latestBlockNumber: number;
+};
+
+/**
+ * @beta
+ *
+ * Get the block number of the latest indexed block
+ *
+ * @param {GetLatestBlockNumberParams} params
+ * @returns {Promise<GetLatestBlockNumberResult>}
+ *
+ * @example
+ * ```ts
+ * import { createThirdwebClient } from "thirdweb";
+ * import { getLatestBlockNumber } from "thirdweb/chainsaw";
+ *
+ * const client = createThirdwebClient({ clientId: "..." });
+ * const { latestBlockNumber } = await getLatestBlockNumber({
+ *  client,
+ *  chainId: 1
+ * });
+ * ```
+ * @chainsaw
+ */
+export async function getLatestBlockNumber(
```

Comment:
This returns the latest indexed block. So the use-case is very specific - if you want to see how up to date chainsaw is. This could be used on the chain page for example.

Reply:
makes sense